cli/app/app.go

```diff
@@ -242,6 +228,21 @@ func Down(c *cli.Context) {
 
 }
 
+// Convenience method to return the appropriate Transformer based on
+// what provider we are using.
+func getTransformer(opt kobject.ConvertOptions) transformer.Transformer {
```

Comment:
yeah - looks like this can't be done because then there would be an import cycle because `transformer.go` would need to import `transformer/kubernetes`, which imports `transformer`. 

```
import cycle not allowed
```